components/automate-ui/src/app/pages/api-token/details/api-token-details.component.ts

```diff
@@ -62,6 +81,31 @@ export class ApiTokenDetailsComponent implements OnInit, OnDestroy {
       .subscribe((id: string) => {
         this.store.dispatch(new GetToken({ id }));
       });
+
+    combineLatest([
+      this.store.select(iamMajorVersion),
+      this.store.select(iamMinorVersion)]).pipe(takeUntil(this.isDestroyed))
+        .subscribe(([major, minor]: [IAMMajorVersion, IAMMinorVersion]) => {
+          this.isMajorV1 = major === 'v1';
+          this.isMinorV1 = minor === 'v1';
+        });
```

Comment:
❓UI-related question: Why are these instance variables and not `Observable<boolean>`, create from a `filter()` that's part of the `pipe()` args above?

Reply:
That was my first instinct, too, especially since they are not used in the ts file. But it makes the template simpler, so we can say
```
<td class="header-column" *ngIf="!isMajorV1 && isMinorV1">
```
instead of
```
<td class="header-column" *ngIf="!(isMajorV1$ | async) && (isMinorV1$ | async)">
```

Reply:
That's true for these two, but if we only ever use this precise combination, we could also make it a `atLeast2p1$` and use it as `*ngIf="atLeast2p1$ | async"`.

Reply:
Yes, I like that. 👍